crates/runtime/src/runtime/outcome.rs

```diff
@@ -1,5 +1,10 @@
 use svm_types::{Gas, ReceiptLog};
 
+/// A wrapper time for `T` that ships with gas information and collected logs.
```

Comment:
`Outcome` is the output of the Transaction. 
It can denote a Succeeding Transaction or a Failing one.
In both cases, the `Outcome` will report the amount of Gas used for executing the transaction and collected logs
while running it.

Reply:
Ok, will add this to the doc.